infrastructure/aws/variables.tf

```diff
@@ -445,31 +445,31 @@ variable "embedding_document_field_name" {
 
 variable "embedding_max_retries" {
   type        = number
-  default     = 10
+  default     = 1
   description = "Number of retries to external embedding services (rate limiting)"
 }
 
 variable "embedding_retry_min_seconds" {
   type        = number
-  default     = 5
+  default     = 120
   description = "Number of seconds to wait before retry to external embedding services (rate limiting)"
 }
 
 variable "embedding_retry_max_seconds" {
   type        = number
-  default     = 120
+  default     = 300
   description = "Maximum number of seconds to wait before retry to external embedding services (rate limiting)"
 }
 
 variable "worker_ingest_min_chunk_size" {
   type        = number
-  default     = 600
+  default     = 1000
   description = "Normal chunk resolution. Minimum size of chunks to be produced by the worker"
 }
 
 variable "worker_ingest_max_chunk_size" {
   type        = number
-  default     = 800
```

Comment:
changing the chunk sizes will impact the quality of search, I know this isn't important right now but worth noting. Do we need to do this?

Reply:
not sure tbh. I increased the chunk size and i got a lot more embedding succeed .. but this was really just be trying stuff at random. ill try without it and the `workers=1`

Reply:
working much better with this suggestion!